superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterBar.tsx

```diff
@@ -263,6 +314,40 @@ const FilterBar: React.FC<FiltersBarProps> = ({
             {t('Apply')}
           </Button>
         </ActionButtons>
+        {isFeatureEnabled(FeatureFlag.DASHBOARD_NATIVE_FILTERS) && (
```

Comment:
Should this be `FeatureFlag.DASHBOARD_NATIVE_FILTERS_SET`?

Reply:
done